src/tools/merssh/command.cpp

```diff
@@ -44,6 +44,18 @@ int Command::executeRemoteCommand(const QString &command)
     // Convert to build engine paths
     QString mappedCommand = remotePathMapping(command);
 
+    // Share host mount points
+    mappedCommand.prepend(QLatin1String("export ") +
+                    QLatin1String("MER_SSH_HOST_SHARED_HOME=") +
```

Comment:
IIRC, we should no longer use QLatin1String

Reply:
This is gone now.